demisto_sdk/commands/generate_docs/generate_playbook_doc.py

```diff
@@ -229,3 +229,15 @@ def get_input_data(input_section: Dict) -> str:
         return default_value.get('simple')
 
     return ''
+
+
+def generate_image_link(playbook_name, custom_image_link):
```

Comment:
please add type hints, and a docstring explaining what `custom_image_link` is and when it should be blank